portafly/src/components/AppLayout.tsx

```diff
@@ -2,7 +2,6 @@ import * as React from 'react'
 import {
   Nav,
   NavList,
-  NavVariants,
```

Comment:
`NavVariants` was deleted in https://github.com/patternfly/patternfly-react/pull/4225